sdk/android/native_api/audio_device_module/audio_device_android.cc

```diff
@@ -125,7 +125,7 @@ rtc::scoped_refptr<AudioDeviceModule> CreateJavaAudioDeviceModule(
       AudioDeviceModule::kAndroidJavaAudio, false /* use_stereo_input */,
       false /* use_stereo_output */,
       jni::kHighLatencyModeDelayEstimateInMilliseconds, std::move(audio_input),
-      std::move(audio_output)
+      std::move(audio_output),
```

Comment:
TODO: This syntax error has been here for a while, even in main, but it looks like this is now being compiled which led to a compile time error. Need to investigate why and if this should be compiled for the Android builds.